data/json/npcs/talk_tags_chat.json

```diff
@@ -69,39 +130,167 @@
+      "Life's been rough. What's keeping you going?",
+      "Got any good news to share?",
+      "Feeling chatty? I'm here for it.",
+      "How's your day been so far?",
+      "Sometimes, a good talk can make a day better. What's up?",
+      "Tell me about your last adventure.",
+      "What's been your biggest challenge recently?",
+      "Let's take a mental break. Got anything fun to share?",
+      "Ever wonder how we ended up here?",
+      "Heard any good rumors?",
+      "Let's swap stories. What's yours?",
+      "Anything you're looking forward to?",
+      "Any new discoveries?",
+      "What's the strangest thing you've encountered?",
+      "Ever feel like there's still some good out there?",
+      "Let's chat about something light. What's up?",
+      "Got any good stories from the road?",
+      "What's a goal you're working towards?",
+      "How do you stay positive?"
     ]
   },
   {
     "type": "snippet",
     "category": "<thoughtfulchitchat>",
-    "//": "General chit chat messages for NPCs having big thoughts.  Tends to be more about establishing game lore.",
+    "//": "General chit chat messages for NPCs having big thoughts. Tends to be more about establishing game lore.",
     "text": [
-      "I just can't believe it's over.  I keep running my head back to the days it all fell apart.  The riots.  The lies.  The psychos.  It never really felt like it was going to go like this.",
-      "You ever think there's any truth to the crap they were spouting before the world ended?  Mind control drugs in the water, bio-terrorism?  Some of it would make sense, but it seems so far-fetched.  Then again, we're dealing with actual zombies.",
-      "I wonder if I should be getting more religious now, or less.  You know what I'm sayin', <name_g>?",
-      "I been thinkin' about rearranging my gear.  It's a real mess.  Don't wanna go for my weapon and accidentally pull out a granola bar, right?",
-      "You ever wonder why we even bother?  We're all just gonna be zombies eventually anyway.  I mean, not that I'm gonna stop fighting, but what's the damn point?",
+      "I just can't believe it's over. I keep running my head back to the days it all fell apart. The riots. The lies. The psychos. It never really felt like it was going to go like this.",
+      "You ever think there's any truth to the crap they were spouting before the world ended? Mind control drugs in the water, bio-terrorism? Some of it would make sense, but it seems so far-fetched. Then again, we're dealing with actual zombies.",
+      "I wonder if I should be getting more religious now, or less. You know what I'm sayin', <name_g>?",
+      "I been thinkin' about rearranging my gear. It's a real mess. Don't wanna go for my weapon and accidentally pull out a granola bar, right?",
+      "You ever wonder why we even bother? We're all just gonna be zombies eventually anyway. I mean, not that I'm gonna stop fighting, but what's the damn point?",
```

Comment:
```suggestion
      "I just can't believe it's over.  I keep running my head back to the days it all fell apart.  The riots.  The lies.  The psychos.  It never really felt like it was going to go like this.",
      "You ever think there's any truth to the crap they were spouting before the world ended?  Mind control drugs in the water, bio-terrorism?  Some of it would make sense, but it seems so far-fetched.  Then again, we're dealing with actual zombies.",
      "I wonder if I should be getting more religious now, or less.  You know what I'm sayin', <name_g>?",
      "I been thinkin' about rearranging my gear.  It's a real mess.  Don't wanna go for my weapon and accidentally pull out a granola bar, right?",
      "You ever wonder why we even bother?  We're all just gonna be zombies eventually anyway.  I mean, not that I'm gonna stop fighting, but what's the damn point?",
```